### PR TITLE
add landing message #1

### PR DESCRIPTION
**Pull request by OsamaAater**

### CODE REVIEW

**Comment by AttwoodData** [comment]
I am not convinced about your <h1> code. Please convince me.

**Review by OsamaAater — COMMENTED** [review]
I improved the code.

**Review by AttwoodData — APPROVED** [review]
The changes look good!

File: index.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<!DOCTYPE html>
+<h1>
```
[the remark this answers is not in the document]
Comment (AttwoodData) [reply index.html:1]: This looks good!

**Review by AttwoodData — CHANGES_REQUESTED** [review]
Make a change here

**Comment by OsamaAater** [comment]
Changes implemented.

**Review by AttwoodData — APPROVED** [review]
This looks good!

**Review by AttwoodData — APPROVED** [review]

**Comment by AttwoodData** [comment]
Can I close the issue?

**Pull request merged**